server/src/app.js

```diff
@@ -2,10 +2,73 @@
 require('dotenv').config();
 require('./database/database');
 
+const Guild = require("./database/models/Guild");
+const User = require("./database/models/User");
+
+const passport = require("passport");
+const { Strategy } = require("passport-discord");
+
+const { Client, Permissions } = require('discord.js');
+const client = new Client();
+client.login(process.env.BOT_TOKEN);
```

Comment:
where is discord.js coming from? this is the backend there is no discord.js

Reply:
Oh well... I need discord.js to check mutual servers and to convert the permissions integer. This way, I'd need another method...

Reply:
You have to fetch from the discord api like it says in the read me.